src/test/java/bridge/domains/BridgeMakerTest.java

```diff
@@ -0,0 +1,26 @@
+package bridge.domains;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class BridgeMakerTest {
+
+    @DisplayName("다리 크기가 3미만이면 예외를 발생시킨다.")
+    @Test
+    void validateBridgeToLowerBound() {
+        Assertions.assertThatThrownBy(() ->
+                        new BridgeMaker(new BridgeRandomNumberGenerator())
+                                .makeBridge(2))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
```

Comment:
```suggestion
    @DisplayName("다리 크기가 3미만이면 예외를 발생시킨다.")
    @Test
    void validateBridgeToLowerBound() {
        final BridgeMaker bridgeMaker = new BridgeMaker(new BridgeRandomNumberGenerator());

        assertThatThrownBy(() -> bridgeMaker.makeBridge(2))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessage("[ERROR] 다리 길이는 3 이상 20 이하만 가능합니다.");
    }
```
현재 `IllegalArgumentException`이 발생하는 부분이 많은 것 같습니다.
`hasMessage()`로 더 범위를 좁혀주는 것이 좋습니다.
검증하려는 예외만 잡도록 작성하는 것이 좋습니다.

Assertions를 static import하면 더 깔끔해집니다.
Assertion 부분은 최대한 간결하게 알아볼 수 있어야 합니다.

Reply:
hasMessage로 좀 더 구체화해야한다는 걸 인지하지 못했습니다.

또한, 방향 입력과 게임 재시작 입력 예외에 대한 테스트도 별도로 처리하도록 했습니다.